### DISCUSSION TITLE
Doctrine/Inflector

### DISCUSSION THREAD

**Discussion by lucianoghione** [laravel/framework#35111] [Q&A] [answered]
**Laravel version:** 6.20.2

**Question:**
Is it ok to use doctrine/inflector 1.4.3 instead of 2.0.3? Could it cause known laravel bugs?

I need to downgrade from 2.0.3 to 1.4.3 for an external package. I'm asking this because I've seen on StackOverflow that someone said this:

_"Installing the doctrine/inflector:1.4.0 brought me some problems, such as the reflection of name of tables in the database for models."_
Source: https://stackoverflow.com/questions/62399935/correctly-install-a-depenency-from-composer

Hope someone can help me wit this!

**Comment by driesvints** [comment, marked as answer]
Both 2.x and 1.4 should work with Laravel 6: https://github.com/laravel/framework/blob/6.x/composer.json#L22

If you need to downgrade because of an external package than maybe consider removing that package.

**Reply by lucianoghione** [reply]
Thank you for replying!!

The problem with that package is that it's a SDK of a payment processor.

The only thing I'm afraid of is about this: 
"Installing the doctrine/inflector:1.4.0 brought me some problems, such as the reflection of name of tables in the database for models."

**Reply by lucianoghione** [reply]
But it's weird because it's accepted by laravel's composer. I think he was wrong in some way

**Reply by GrahamCampbell** [reply]
`doctrine/inflector:1.4.0` is unlikely what you want. You want `doctrine/inflector:^1.4`. Very different. :)
